tests/unit/test_charm.py

```diff
@@ -119,22 +119,6 @@ def test_on_install(harness):
         assert isinstance(harness.model.unit.status, WaitingStatus)
 
 
-def test_on_install_snap_failure(harness):
```

Comment:
we should have test(s) for failed snap instannation to make sure `juju resolve -all` fixes it.